src/components/AuthForm/ForgotPasswordPage.scss

```diff
@@ -0,0 +1,240 @@
+#forgot-password-page-container {
+	height: 100vh;
+	width: 100vw;
+	background: linear-gradient(45deg, hsla(234, 36%, 55%, 1) 0%, hsla(234, 36%, 55%, 0) 100%),
+		url("../../assets/images/BgGraphics.png") top right/auto 100% no-repeat;
+
+	.container {
```

Comment:
@Ayush-Bharadva  I think I wasn't clear enough in my previous comment: 
the look and feel of forgotPassword form should match that of login and Registration:



![remove extra black background](https://github.com/Ayush-Bharadva/fitnesstracker/assets/30732258/203c87c8-2d07-47e4-a003-6d7b386cbba8)


1. Remove black background.
2. The color of the header font should match that of login and registration
3. The email field should have a label with font color similar to what we have in login and registration.
4. The CTA button verify should have the same color as CTA button in login and registration!

I know you are just starting out but AS A FRONTEND DEV it is your responsibility to make sure UI/UX feels cohesive.